content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,148 @@
+to view all products under the category they select. To do so, go to :menuselection:`eCommerce -->
+Products --> (Your product) --> Sales` and select the :guilabel:`Categories` you want under
+:guilabel:`eCommerce Shop`.
+
+.. note::
+   A single product can appear under multiple eCommerce categories.
+
+When your product is configured, go to the :guilabel:`Shop` page of your website and click on
+:menuselection:`Edit --> Customize`. In the :guilabel:`Categories` option, you can either enable a
+menu on the :guilabel:`Left`, on the :guilabel:`Top`, or both. The :guilabel:`Collapsable Category
+Recursive` button renders the *left* category menu collapsable.
+
+.. image:: catalog/catalog-panel-categories.png
+   :align: center
+   :alt: Categories options for your eCommerce website
+
+Browsing
+--------
+
+The eCommerce category is the first tool to organize and split your products. However, if you need
+an extra level of categorization in your catalog, you can activate various **filters** such as
+attributes or sort-by search.
+
+Attributes
+~~~~~~~~~~
+
+Attributes refer to **characteristics** of a product such as **color** or **material**, whereas
+variants are the different combinations of attributes. Attributes and variants can be found under
+:menuselection:`eCommerce --> Products --> (Your product) --> Attributes & Variants`.
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/products/variants`
+
+.. image:: catalog/catalog-attributes.png
+   :align: center
+   :alt: Attributes and variants of your product
+
+To enable **attributess** filtering, go to :menuselection:`Shop page --> Edit --> Customize` and
+select either :guilabel:`Left`, :guilabel:`Top`, or both. Additionally, you can also enable
+:guilabel:`Price Filtering` to enable price filters.
+
+.. note::
+   :guilabel:`Price Filter` works independently from **attributes** and therefore can be enabled on
+   its own if desired.
+
+.. tip::
+   You can use **attribute filters** even if you do not work with product variants. When adding
+   attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
+   create variants if no combination is possible.
+
+Sort-by search
+~~~~~~~~~~~~~~
+
+It is possible to allow the user to manually **sort the catalog** using the search bar. From
+:menuselection:`Shop page --> Edit --> Customize`, you can enable or disable the :guilabel:`Sort-By`
+option as well as the :guilabel:`Layout` button. You can also select the :guilabel:`Default Sort` of
+the :guilabel:`Sort-By` button. The default sort applies to *all* categories.
+
+The **sorting** options are:
+
+- Featured
+- Newest Arrivals
+- Name (A-Z)
+- Price - Low to High
+- Price - High to Low
+
+In addition, you can **manually edit** the order of a product by going to :menuselection:`Shop page
+--> Edit` and clicking on the product. Under the :guilabel:`Product` section of the
```

Comment:
go to :menuselection:Wesbite app --> Shop click on :guilabel:Edit and then select the product.

Reply:
please also update for the other paths below :)